File: es_09.cpp
// 9. Inserire da tastiera un numero N, calcolare la sua tabellina dall’1 al 10.

#include <iostream>
using namespace std;

int main() {
    int n;

    cout << "Inserisci un numero: ";
    cin >> n;

    for (int i = 1; i <= 10; i++) {
        cout << n * i << endl;
    }

    return 0;
}

File: Python/es_09.py
from itertools import count

# 9. Inserire da tastiera un numero N, calcolare la sua tabellina dall’1 al 10.

while True:
    n = int(
        input('Inserire un numero N > 0: ')
    )
    
    if n > 0: break

for i in count (start=1, step=1):
    print(
        f'{n * i}', end=', ' if i < 10 else '\n'
    )
    
    if i >= 10: break